src/functions/sendslack/timebankdata.ts

```diff
@@ -0,0 +1,84 @@
+const fetch = require('node-fetch');
+import { PersonDto, TimeEntryTotalDto } from "./schema";
+
+/**
+ * 
+ * @returns Working hours and Persons information
+ */
+
+const timebankData = async () => {
+
+  /**
+   * gets data from TimeBank API
+   * 
+   * @returns filtered persons data what we need.
+   */
+
+  const getData = async (): Promise<PersonDto[]> => {
+    const Data = (await
+      (await fetch('https://time-bank-api.metatavu.io/timebank/persons')
+      ).json());
```

Comment:
You should include open api generator to this project. I can help you with that. It will add types for this project and you can get the actual return types for request responses.